code/obj/item/tool/electronics.dm

```diff
@@ -415,30 +416,33 @@
 	var/viewstat = 0
```

Comment:
could we get some help about the harm intent added to the desc, so people don't freak out about puttting it in their backpacks